lib/tests/main.cc

```diff
@@ -12,23 +16,28 @@
 #include <leatherman/logging/logging.hpp>
 #endif
 
-int main(int argc, char* const argv[]) {
+int main(int argc, const char** argv) {
 #ifdef ENABLE_LOGGING
+    leatherman::logging::setup_logging(boost::nowide::cout);
     leatherman::logging::set_level(leatherman::logging::log_level::debug);
 #endif
 
-    // configure the Catch session and start it
-    Catch::Session test_session;
+    // Create the Catch session, pass CL args, and start it
+    Catch::Session test_session {};
```

Comment:
What's the difference between:
```C++
Catch::Session test_session;
```
and
```C++
Catch::Session test_session {};
```
isn't the constructor called in the first case or what?